.gitattributes

```diff
@@ -0,0 +1,36 @@
+#source files
```

Comment:
I do not think we need this file as all we'll have here are java, md, xml and probably few sh files

Reply:
fair, enough. I figured it didn't hurt to specify file extensions that don't currently exist, but might in the future. I have no problems reducing it down though to the currently relevant extensions. 

Also, thoughts on `NOTICE.md`? There are a huge list of third-party references that aren't used here